src/frontend/js/components/DashboardItem/_styles.scss

```diff
@@ -150,4 +131,68 @@
       margin: 0;
     }
   }
+
+  &__runs {
+    position: relative;
+  }
+
+  &__loading {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background: rgba(255, 255, 255, 0.8);
```

Comment:
This color should be picked from a theme value. Why not use modal background-overlay?

Reply:
The `modal__overlay` is using position: fixed, and not centering flex items. And the semantic seemed strange to me. I replaced the color with a themed, one. WDYT ?